build.gradle

```diff
@@ -181,11 +178,11 @@ pluginBundle {
 publishing {
     publications {
         libertyGradle (MavenPublication) {
-            artifactId = archivesBaseName
+            artifactId = 'liberty-gradle-plugin'
```

Comment:
Why aren't we using `base.archivesName` here?

Reply:
maybe if the `base` plugin is applied...it will work? See previous comment.

Reply:
it works without and with apply base plugin. applied base plugin to resolve confusion